src/services/RegisterUserService.js

```diff
@@ -1,25 +1,57 @@
 const User = require("../models/User");
 
-const jwtGenerate = require("../utils/jwtGenerate");
+const Response = require("../utils/responses");
+
+const JwtManager = require("../utils/jwtManager");
 
 module.exports = async (req, res) => {
-  const { email } = req.body;
+  const jwt = new JwtManager();
+
+  const response = new Response(res);
+  const { entities } = response;
 
   try {
+    const { name, picture, email, password } = req.body;
+
     if (await User.findOne({ email }))
-      return res.status(400).json({ error: "User already exists" });
+      return response
```

Comment:
the best status for this case and 409, can you change that please?

Reply:
Right